apps/builder/src/features/editor/constants.ts

```diff
@@ -1 +1,5 @@
 export const headerHeight = 56;
+
+export enum LOCAL_STORAGE_KEY {
+  LEFT_SIDEBAR_LOCKED = "LEFT_SIDEBAR_LOCKED",
+}
```

Comment:
To match project coding style I'd prefer having a single non-maj const if possible:

```ts
export const leftSidebarLockedStorageKey = "isLeftSidebarLocked"
```